src/main/fc/cli.c

```diff
@@ -3079,14 +3081,15 @@ static void cliVersion(char *cmdline)
 {
     UNUSED(cmdline);
 
-    cliPrintLinef("# %s / %s (%s) %s %s / %s (%s)",
+    cliPrintLinef("# %s / %s (%s) %s %s / %s (%s) %s",
```

Comment:
Sorry, I attached my previous comment to the wrong line, it should really have gone here.

Reply:
Done, rebased and squashed. It now looks like: 
```
# version
# Betaflight / SPRACINGF3 (SRF3) 3.2.0 Sep 10 2017 / 22:09:06 (c73ad66b0) MSP API: 1.36
```